src/acq2.c

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Codethink Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "acq.h"
+
+void bl_acq_init(uint32_t clock)
+{
+	(void)clock;
```

Comment:
We can `#include "util.h"` and

```suggestion
	BL_UNUSED(clock);
```

For a bit more explanation.  Ditto for the other voided parameters.

Reply:
Good point, will change.

Reply:
Hmm, just leave it for now then!

Reply:
Fixed